flink-libraries/flink-state-processing-api/src/main/java/org/apache/flink/state/api/output/BoundedStreamTask.java

```diff
@@ -67,7 +67,7 @@
 	@Override
```

Comment:
For the future. It looks like `StreamOperatorWrapper`  should have been introduced in a separate commit to the changes of `endInput` logic.

Reply:
I don't think I understand. Is that the comment (https://github.com/apache/flink/pull/10151#discussion_r369234977)?